tests/app/main/test_frameworks.py

```diff
@@ -182,7 +182,7 @@ def test_email_sent_when_interest_registered_in_framework(self, send_email, data
             'You started a G-Cloud 7 application',
             'do-not-reply@digitalmarketplace.service.gov.uk',
             'Digital Marketplace Admin',
-            ['digital-outcomes-and-specialists-application-started']
+            ['g-cloud-7-application-started']
```

Comment:
Is is also worth checking that `render_template` got called with the correct arguments when the email is templated?

Reply:
So I tried to mock out `render_template` and _everything_ broke. I spent about 5 minutes looking into it and gave up. I could put some more effort in as it does feel like a sensible thing to test. 🤔

Reply:
_Everything_ broke? That is a lot....

Reply:
There is this `wraps` thing that I bumped into a the weekend. I've not used it, but I think it allows you to mock an object without affecting it's functionality, so you can effectively spy on it. Might be useful here? https://docs.python.org/3/library/unittest.mock.html